main/src/main/java/cgeo/geocaching/filters/core/GeocacheFilterType.java

```diff
@@ -23,6 +23,7 @@ public enum GeocacheFilterType {
     DIFFICULTY_TERRAIN_MATRIX("difficulty_terrain_matrix", R.string.cache_filter_difficulty_terrain_matrix, R.string.cache_filtergroup_details, DifficultyTerrainMatrixGeocacheFilter::new),
     RATING("rating", R.string.cache_filter_rating, R.string.cache_filtergroup_details, RatingGeocacheFilter::new),
     STATUS("status", R.string.cache_filter_status, R.string.cache_filtergroup_basic, StatusGeocacheFilter::new),
+    IndividualRoute("individualroute", R.string.cache_filter_individualroute, R.string.cache_filtergroup_userspecific, IndividualRouteGeocacheFilter::new),
```

Comment:
all the other filter names are written in caps

Reply:
You are right, interesting that codacy didn't complain. I thought there was a "Enum names must be uppercase" check somewhen...